recipes/tcsh/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "tcsh" %}
+{% set version = "6.20.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tcsh-org/tcsh.git
+  git_tag: TCSH6_20_00
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - ncurses
```

Comment:
Needs to be [pinned to `ncurses 5.9`]( https://github.com/conda-forge/conda-forge.github.io/blob/5cbfa5719f0fc5b67edb3fa13f3d7f0293766f6f/scripts/pin_the_slow_way.py#L74 ).